crates/autopilot/src/run_loop.rs

```diff
@@ -176,6 +177,12 @@ impl RunLoop {
                     .find(|auction_order| &auction_order.uid == order_id);
                 match auction_order {
                     Some(auction_order) => {
+                        for fee_policy in auction_order.fee_policies.clone() {
```

Comment:
I would push the fee policy only if the solution is a winner solution. There is a code below that similarly checks for winner: 

`let is_winner = solutions.len() - index == 1;`

Otherwise, if two or more solvers return solutions containing the same order, we will end up with multiple same fee policies for the same <auction_id, order_uid> and based on the current implementation those would be different entries in the table with different `application_order` column AFAIS.

Reply:
But currently, we are working with the latest solution.

Reply:
Oh, my bad. All good.